README.md

```diff
@@ -27,13 +27,17 @@ export default defineConfig({
     // I have `ctx`! which is provided by the above
     return;
   },
-};
+});
 ```
 
 Run `malcolm`
 
 ### Development
 
-Working in this directory?Clone and `npm run start:dev`
+1. Clone
+1. Create a `proxy.config.ts` in the `packages/proxy` directory
```

Comment:
Does it need to be in `packages/proxy`? I thought it will find it based on your `cwd`? Quite possible that I am WRONG

Reply:
It worked for me in `packages/proxy`. Let me mess around.

Reply:
I think [this line](https://github.com/nickhudkins/malcolm/blob/mainline/packages/proxy/src/program.ts#L14) resolves a relative path to the package itself. But to your point, it can just be a CLI arg.